packages/components/src/Select/Select.tsx

```diff
@@ -133,26 +122,39 @@ export const Select = ({
           {state.selectedItem ? state.selectedItem.rendered : placeholder}
         </Box>
         {state.isOpen && !disabled ? (
-          <ArrowUp size={16} className={iconClassName} />
+          <Box
+            as={ArrowUp}
+            size={16}
+            css={{ fill: disabled ? 'disabled' : 'text' }}
```

Comment:
Was genau macht das? 🤔

Reply:
icon color

Reply:
We dont need this because if disabled then ArrowUp is never visible. Uuups